python/sqlflow_submitter/db_writer/hive.py

```diff
@@ -96,9 +96,9 @@ def write_hive_table(self):
             hdfs_envs.update({"HADOOP_USER_NAME": self.hdfs_user})
         if self.hdfs_pass != "":
             hdfs_envs.update({"HADOOP_USER_PASSWORD": self.hdfs_pass})
-        cmd_str = "hdfs dfs -mkdir -p %s/%s/" % (hdfs_path, self.table_name)
+        cmd_str = "hdfs dfs -fs hdfs://%s -mkdir -p %s/%s/" % (namenode_addr, hdfs_path, self.table_name)
```

Comment:
We are not using `namenode_addr` currently because, in some deployment, the HDFS client needs to be configured to connected to multiple namenodes so that this deployment is Highly Available.

Reply:
Maybe if namenode_addr is set (not empty string), then add `-fs` else, the command keeps `"hdfs dfs -mkdir -p %s/%s/"` to support both multi namenode setup and single namenode setup.